internal/api/mfa.go

```diff
@@ -40,6 +43,11 @@ type EnrollFactorResponse struct {
 	Type         string     `json:"type"`
 	FriendlyName string     `json:"friendly_name"`
 	TOTP         TOTPObject `json:"totp,omitempty"`
```

Comment:
i tried enrolling an SMS factor and the response i got was:

```json
{
  "id": "b77da682-370d-46c1-80a4-e505b9ece9fb",
  "type": "sms",
  "friendly_name": "",
  "totp": {
    "qr_code": "",
    "secret": "",
    "uri": ""
  },
  "phone_number": "123456789"
}
```

are the TOTP fields necessary to be returned here? if not, we need to add `omitempty` to the fields in the `TOTPObject` so that it's returned like this instead: `"totp":{}`

Reply:
I suppose everything in TOTPObject needs to be `omitempty` as well?